### DISCUSSION TITLE
Please tell me your favorite pokemon!!

### DISCUSSION THREAD

**Discussion by Masashi-ABE** [NAIST-SE/AttractingContributors#208] [General]
https://github.com/Masashi-ABE/AttractingContributors-Pokemon

**Comment by TakumiHiraide** [comment]
Please write wherever you want to go now!!!
https://github.com/TakumiHiraide/Where-do-you-want-to-go-

**Reply by TakumiHiraide** [reply]
I added Pokemon to your repository so please help me!!
